### PR TITLE
vers 0.6.2; updates project Gemfile.lock to latest activerecord, activesupport and grape gems

### PR DESCRIPTION
**Pull request by dylanfareed**
Also includes updates to/for rubocop and rspec

### CODE REVIEW

File: lib/gris/seed_loader.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ def initialize(file_path)
     end
 
     def load_seed
-      fail("Seed file '#{@seed_file}' does not exist.") unless File.file?(@seed_file)
+      raise("Seed file '#{@seed_file}' does not exist.") unless File.file?(@seed_file)
```
Comment (starsirius) [inline lib/gris/seed_loader.rb]: No more `fail`s!!! (It's a success!!)

**Comment by starsirius** [comment]
An unrelated question: Does gris support generating namespaced endpoints/presenters now, e.g. `Admin::V1::RootEndpoint`?

**Comment by dylanfareed** [comment]
Not generating from scratch but we have been doing that a ton manually.

**Comment by dylanfareed** [comment]
Released! Thanks for the merge @starsirius

**Comment by starsirius** [comment]
Woohoo! :tada:

**Pull request merged**